make/photon/portal/nginx.conf

```diff
@@ -1,13 +1,21 @@
 
-worker_processes  1;
+worker_processes auto;
```

Comment:
What's the rationale behind this change?

Reply:
`auto` will improve the performance when input traffic is large.

Reply:
ok